CMakeLists.txt

```diff
@@ -1,18 +1,20 @@
 cmake_minimum_required(VERSION 2.8.0)
 
 project(express-cpp)
-set(CMAKE_BUILD_TYPE Debug)
-enable_testing()
+
+option (UNIT_TESTS "Enable Unit Tests" OFF)
 
 set (VERSION_MAJOR 0)
 set (VERSION_MINOR 1)
 set (VERSION_PATCH 0)
 
 set (VERSION_STRING_INSTALL "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 
-add_definitions(--std=c++11)
```

Comment:
Why do you remove the C++11 flag ? When doing so, it still compiles on travis as I'm using new compilers with this setting as default, but on OS X this is not default yet and fails, so I need it.